### DISCUSSION TITLE
Expected — Waiting for status to be reported

### DISCUSSION THREAD

**Discussion by gponty** [community/community#25720] [Actions]
<p>Hi,</p>
<p>Why my task stuck like this :</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/71c113bd6d5f80d52e61ff733b57da192721b28a.png" data-download-href="/uploads/short-url/gejCr7FK6fyGcJlDboW93NlTP06.png?dl=1" title="Capture du 2020-03-31 13-43-25.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/34276317/181092823-2788d581-d93b-49b5-9440-f27af8a4d398.png" alt="Capture du 2020-03-31 13-43-25.png" data-base62-sha1="gejCr7FK6fyGcJlDboW93NlTP06" width="690" height="331" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/7/71c113bd6d5f80d52e61ff733b57da192721b28a_2_690x331.png, https://aws1.discourse-cdn.com/github/original/2X/7/71c113bd6d5f80d52e61ff733b57da192721b28a.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/7/71c113bd6d5f80d52e61ff733b57da192721b28a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/71c113bd6d5f80d52e61ff733b57da192721b28a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture du 2020-03-31 13-43-25.png</span><span class="informations">750×360 34.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thank you</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/gponty">@gponty</a> ,</p>
<p>Please check with the below steps:</p>
<ol>
<li>
<p>Open Checks tab to check whether the Required check ( <strong>Deploiement master</strong> ) is really executed.</p>
</li>
<li>
<p>If the required check is not really executed, you can open the <strong>Settings</strong> tab of your repository and navigate to <strong>Branches</strong> &gt; <strong>Branch protection rules</strong> , open the rules you have set, find the option " <strong>Require status checks to pass before merging</strong>" to check if you have set " <strong>Deploiement master</strong>" as " <strong>Required</strong>" in the list, disable it.</p>
</li>
</ol>
<p>If you have marked some checks as " <strong>Required</strong>" for PR in your repository, every time when you run checks for a PR, the required checks will be listed in the PR. If a required check is not included in the actually executed checks for the PR (that means no job in the workflows which run for the PR can match this required check), this check will stuck here to prevent the PR from being merged.</p>
<p>In this situation, you need to disable this required check in the related branch rules. In most cases, this is because you had deleted the job or changed the job (renamed it) from the workflows, but did not to update the branch rules.</p>
<p>For example:</p>
<p>I had set the two checks " <strong>Build</strong>" and " <strong>Trigger Azure pipeline run</strong>" as " <strong>Required</strong>". But the job " <strong>Trigger Azure pipeline run</strong>" was not actually executed because I had removed it from the workflow, it stuck here.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/604ee14608be7c4fe784352228aeaa5ee289d422.png" data-download-href="/uploads/short-url/dJYPSo9VI5GXTkPE3otkFO6Ju2S.png?dl=1" title="check_runs.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181092824-a0bf66e1-f578-4d15-93f2-567a9e5ef30a.png" alt="check_runs.PNG" data-base62-sha1="dJYPSo9VI5GXTkPE3otkFO6Ju2S" width="690" height="222" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/604ee14608be7c4fe784352228aeaa5ee289d422_2_690x222.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/604ee14608be7c4fe784352228aeaa5ee289d422_2_1035x333.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/604ee14608be7c4fe784352228aeaa5ee289d422.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/604ee14608be7c4fe784352228aeaa5ee289d422_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">check_runs.PNG</span><span class="informations">1171×378 33.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So I go to update the branch rules to disable " <strong>Trigger Azure pipeline run</strong>".</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8db4ddbeb05a5fabaa2c9dd3440f1c0fcaea4991.png" data-download-href="/uploads/short-url/kdAPin6Rvmh5eRih9oa4pGcOYa5.png?dl=1" title="rules.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181092826-6f062f63-c1c4-4131-8f23-7669a160ebe1.png" alt="rules.PNG" data-base62-sha1="kdAPin6Rvmh5eRih9oa4pGcOYa5" width="690" height="247" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8db4ddbeb05a5fabaa2c9dd3440f1c0fcaea4991_2_690x247.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/8db4ddbeb05a5fabaa2c9dd3440f1c0fcaea4991_2_1035x370.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/8db4ddbeb05a5fabaa2c9dd3440f1c0fcaea4991_2_1380x494.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8db4ddbeb05a5fabaa2c9dd3440f1c0fcaea4991_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">rules.PNG</span><span class="informations">1749×627 72.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Then reopen/refresh the page of the PR, all checks were passed, and I can merge the PR.</p>
<p><img src="https://user-images.githubusercontent.com/46307996/181092827-afbb442d-192e-4b19-8c54-71f714d0764d.png" alt="pass.PNG" data-base62-sha1="evnjT7WXPVGnMP4ygWkM5I6d83y" width="690" height="381"></p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/gponty">@gponty</a> ,</p>
<p>How are things going? Is my above suggestion helpful to you?</p>
<p>If you have any question about this ticket, feel free to tell me.</p>

**Comment by gponty** [comment]
<p>Thank you <a class="mention" href="/brightran">@brightran</a> ! That’s help me !</p>
<p>Take care</p>

**Comment by Koshak118** [comment]
<p><a class="mention" href="/brightran">@brightran</a> </p>
<p>Hi, I am having the same symptom of never-ending required check, but related to PR path filters configured in the appropriate Azure Pipelines YAML.</p>
<p>Azure guys <a href="https://github.com/MicrosoftDocs/azure-devops-docs/issues/3210#issuecomment-464836540" rel="nofollow noopener">state it should return “Neutral” status</a> when the path fiter does filter out the build, but for me it gets stuck forever</p>
<p>Could you please advise?</p>

**Comment by johnhamelink** [comment]
<p>Hey folks,</p>
<p>We’re getting the same issue on our organization - we average about 20 pull requests a day and are experiencing this every 2 days or so. It only happens with the dependabot check: it seems that (for us at least) this comes up quite frequently, and disabling branch protection is not a particularly constructive solution.</p>
<p><a class="mention" href="/brightran">@brightran</a> if you would like some extra data to analyse this issue, we’d be happy to help.</p>
<p>Best,<br>
JH</p>

**Comment by petermetz** [comment]
<p>Same issue here, dependabot seems to be the X factor.<br>
I tried to delete the PR description thinking that maybe that will kick it into gear but it didn’t help either.</p>
<p>This is the specific PR where it can be observed: <a href="https://github.com/hyperledger/cactus/pull/759" class="inline-onebox" rel="noopener nofollow ugc">chore(deps): bump y18n from 4.0.0 to 4.0.1 in /packages/cactus-cockpit by dependabot · Pull Request #759 · hyperledger/cactus · GitHub</a><br>
<a class="mention" href="/brightran">@brightran</a></p>

**Comment by mur-me** [comment]
<p>Thanks for this solution &lt;3</p>

**Comment by coreysan** [comment]
<p>Removing the required branch rule from GitHub fixed this for me :+1:</p>

**Comment by SamTov** [comment]
Occasionally this can be caused by an error in the test yaml file. You will need to look at the checks tab in order to identify if this is the case.

**Reply by jzabroski** [reply]
The Checks tab no longer exists... it would be great if this answer was replaced with something that matches the current UI. My CI builds for PRs have been broken for about a year now. It's difficult to find time to investigate why this happened.
